.well-known/brave-rewards-verification.txt

```diff
@@ -1,4 +0,0 @@
-This is a Brave Rewards publisher verification file.
-
-Domain: tomkadwill.com
-Token: c94e7c6a2ccab74254ede36ff216178af22aa1534dddab838df67c7f364ab780
```

Comment:
Hah! What the rewards program like?

Reply:
Didn't get it working 😄. I linked my YouTube account but haven't earned anything